static/gsApp/views/subscriptionPage/seerAutomationAlert.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from '@emotion/styled';
+
+import {Alert} from 'sentry/components/core/alert';
+import {Button} from 'sentry/components/core/button';
+import Link from 'sentry/components/links/link';
+import {IconClose} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types/organization';
+import useDismissAlert from 'sentry/utils/useDismissAlert';
+import {useLocation} from 'sentry/utils/useLocation';
+
+interface SeerAutomationAlertProps {
+  organization: Organization;
+}
+
+export default function SeerAutomationAlert({organization}: SeerAutomationAlertProps) {
+  const location = useLocation();
+  const isCheckoutRedirect = !!location.query.referrer; // only show this alert when redirected to this page, such as from the checkout page
```

Comment:
i don't think looking at the presence of `referrer` alone is sufficient here

there are a couple in-product places where we redirect to this page and provide a referrer that aren't from checkout

Reply:
instead, we can modify the redirect in `submitCheckout` to pass a query parameter when the user has just bought seer, ie. 
```js
hasBoughtSeer = // whatever logic to indicate a user just bought seer
normalizeUrl(
        `/settings/${organization.slug}/billing/overview/?referrer=${referrer}&show_seer_automation_alert={hasBoughtSeer}`
      )
```

Reply:
Done